packages/react-admin/src/mui/layout/Layout.js

```diff
@@ -97,14 +97,7 @@ const Layout = ({
                         open && classes.contentShift
                     )}
                 >
-                    <AdminRoutes
```

Comment:
if the layout changes, this is a BC break, and it should be documented in the UPGRADE guide. The theming doc needs an update, too.

Reply:
Done